componentapiexample/src/main/java/net/gini/android/vision/component/MainActivity.java

```diff
@@ -205,12 +204,12 @@ private void startGiniVisionStandard() {
                     @Override
                     public void permissionGranted() {
                         // NOTE: on Android 6.0 and later the camera permission is required before checking the requirements
-                        final RequirementsReport report = GiniVisionRequirements.checkRequirements(
-                                MainActivity.this);
-                        if (!report.isFulfilled()) {
-                            showUnfulfilledRequirementsToast(report);
-                            return;
-                        }
+//                        final RequirementsReport report = GiniVisionRequirements.checkRequirements(
```

Comment:
Same here!

Reply:
Same as above and will also update it in the Screen API example app's `MainActivity`.